components/ui/InfiniteScroll.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <section>
+    <div
+      v-for="(item, index) in items.slice(0, lastShown)"
+      :id="index + '-' + keyGenerator(item)"
+      :key="keyGenerator(item)"
+      class="scrollable"
+    >
+      <slot v-bind="item" />
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import ScrollSectionsMixin from '~/mixins/scrollBetweenSections'
+
+@Component
+export default class extends Mixins(ScrollSectionsMixin) {
```

Comment:
Naming component classes can help with DX by making them easier to distinguish with [Vue.js devtools](https://github.com/vuejs/vue-devtools):

```suggestion
export default class InfiniteScroll extends Mixins(ScrollSectionsMixin) {
```

This would result in:

<img width="549" alt="Screenshot 2020-11-11 at 16 46 13" src="https://user-images.githubusercontent.com/22047320/98832706-6b7ea900-243d-11eb-83e1-b24b37dbe621.png">

Reply:
This is a project convention. If that should change, we have to change the whole project.

Reply:
Is there a discussion issue related to this? I'm afraid I couldn't find one if that's the case.

Otherwise I'd open an issue to have a discussion about pros and cons that this change could convey.

Reply:
Ok, open it.

Reply:
Opened here: https://github.com/Qiskit/qiskit.org/issues/1103